README.md

```diff
@@ -12,7 +12,7 @@ First install the system packages:
 
 ```bash
 sudo apt install xxd cmake ninja-build
-pip install lit wget
+pip install lit wget numpy Pillow
```

Comment:
Please rebase. The python requirement has been converted into a list in https://github.com/AmbiML/iree-rv32-springbok/pull/2.